<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Sobhanaltop" name="com.sampleMuvi.runner.TestRunner" tests="1" failures="1" timestamp="2 Aug 2021 09:06:55 GMT" time="0.000" errors="0">
  <testcase name="feature" time="0.000" classname="com.sampleMuvi.runner.TestRunner">
    <failure type="java.lang.RuntimeException" message="java.lang.IllegalArgumentException: Not a file or directory: D:\GVS_TITAN_AUTOMATION\sample-Muvi\.src\test\java\com\sampleMuvi\feature">
      <![CDATA[java.lang.RuntimeException: java.lang.IllegalArgumentException: Not a file or directory: D:\GVS_TITAN_AUTOMATION\sample-Muvi\.src\test\java\com\sampleMuvi\feature
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:143)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:426)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.IllegalArgumentException: Not a file or directory: D:\GVS_TITAN_AUTOMATION\sample-Muvi\.src\test\java\com\sampleMuvi\feature
	at cucumber.runtime.io.FileResourceIterator$FileIterator.<init>(FileResourceIterator.java:54)
	at cucumber.runtime.io.FileResourceIterator.<init>(FileResourceIterator.java:20)
	at cucumber.runtime.io.FileResourceIterable.iterator(FileResourceIterable.java:19)
	at cucumber.runtime.model.CucumberFeature.loadFromFeaturePath(CucumberFeature.java:103)
	at cucumber.runtime.model.CucumberFeature.load(CucumberFeature.java:54)
	at cucumber.runtime.model.CucumberFeature.load(CucumberFeature.java:34)
	at cucumber.runtime.RuntimeOptions.cucumberFeatures(RuntimeOptions.java:239)
	at cucumber.api.testng.TestNGCucumberRunner.getFeatures(TestNGCucumberRunner.java:85)
	at cucumber.api.testng.TestNGCucumberRunner.provideFeatures(TestNGCucumberRunner.java:94)
	at cucumber.api.testng.AbstractTestNGCucumberTests.features(AbstractTestNGCucumberTests.java:29)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:117)
	... 20 more
]]>
    </failure>
  </testcase> <!-- feature -->
</testsuite> <!-- com.sampleMuvi.runner.TestRunner -->
